sympy/stats/frv.py

```diff
@@ -339,7 +339,7 @@ def conditional_space(self, condition):
                 for key, val in self._density.items() if domain._test(key))
         return FinitePSpace(domain, density)
 
-    def sample(self, size=()):
+    def sample(self, size=(1,), library='scipy'):
```

Comment:
What is the purpose of substituting `()` with `(1,)` or `1` ? The tuple notation should represent the dimensions of an N-dimensional array you want to be returned.